ccd-definition/ComplexTypes/C21Order.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "C21Order",
+    "ListElementCode": "orderTitle",
+    "FieldType": "Text",
+    "ElementLabel": "Order title",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "C21Order",
+    "ListElementCode": "orderDetails",
+    "FieldType": "TextArea",
+    "ElementLabel": "Order details",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "orderDate!=\"*\""
```

Comment:
why field show condition?

Reply:
don't want to display order details in the tab view